.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Is this necessary? It is already git ignored. With this set, it would mean that locally if we see a vulun, running `npm audit` to see what it is won't work, since there won't be a lock file.

Reply:
Unless you blow it away regularly locally, ignoring it only means your lock will be in a different state than mine and anyone else who works on the package.  IMO it is better to run `npm i --package-lock-only && npm audit && rm package-lock.json` when you want the audit than risk the out of sync behavior on accident.

Reply:
Sure, but doesn't that argument apply to `node_modules` already? What makes sure both of ours stays in sync?

Reply:
I guess you can make that argument.  I guess it is just a more common workflow to clear out `node_modules` and if you forget the lock, which is very commonly committed to to projects, you will succeed in never getting the new updates.  Much more error prone IMO.

Reply:
I guess, but on that argument, then this module should also assure that `npm-shrinkwrap.json` or the `yarn.lock` files are also not created? It sounds like your argument is that the user will half-know their tooling (i.e. they are aware their tooling creates a `node_modules` folder, but not that it is making a `package-lock.json` file) so we should at least try to help out for that use-case. Unless I'm misunderstanding the use-case for this.

Reply:
That is probably also a good idea, although I don't use yarn and do not know a way to fully disable, and [the docs also don't mention it](https://yarnpkg.com/lang/en/docs/yarn-lock/).  As for a shrinkwrap, it is much less commonly used and not created by default, it is much less of an issue.

The problem with `package-lock.json` is that it is created by default without that setting on every install, and you have to be very vigilant to remove it and test before testing to ensure you wont just break in CI.